packages/serverpod/lib/src/server/session.dart

```diff
@@ -186,7 +186,12 @@ abstract class Session implements DatabaseAccessor {
     }
 
     if (Features.enableConsoleLogging) {
-      logWriters.add(StdOutLogWriter(session));
+      var logFormat = session.serverpod.config.sessionLogs.consoleLogFormat;
+      var consoleLogger = switch (logFormat) {
```

Comment:
```suggestion
      final logFormat = session.serverpod.config.sessionLogs.consoleLogFormat;
      final consoleLogger = switch (logFormat) {
```

Even though we don't have that linter here yet? We might just slowly transition (else I would think personally it's hard to figure out which style to use where).